docs/samples/management-api-v2-walkthrough/2-assets.md

```diff
@@ -0,0 +1,83 @@
+# Creating an Asset
+
+## Old plain JSON Schema
+
+```json
+{
+  "asset": {
+    "id": "asset-id",
+    "properties": {
+      "name": "asset-name",
+      "description": "asset-description",
+      "version": "0.0.1",
+      "contenttype": "application/json"
+    }
+  },
+  "dataAddress": {
+    "type": "asset-address-type",
+    "keyName": "asset-key-name"
+  }
+}
+```
+
+## New JSON-LD Document
+
+> Please note: In our samples, properties **WILL NOT** be explicitly namespaced, and internal nodes **WILL NOT** be typed, relying on `@vocab` prefixing and root schema type inheritance respectively.
+
+```json
+{
+  "@context": {
+    "@vocab": "https://w3id.org/edc/v0.0.1/ns/"
+  },
+  "@type": "AssetEntryDto",
+  "asset": {
+    "@id": "asset-id",
+    "properties": {
+      "name": "asset-name",
+      "description": "asset-description",
+      "version": "0.0.1",
+      "contenttype": "application/json"
+    },
+    "privateProperties": {
+      "pvt-prop-1": "prt-prov-val-1",
+      "pvt-prop-2": "prt-prov-val-2"
+    }
+  },
+  "dataAddress": {
+    "type": "asset-address-type",
+    "keyName": "asset-key-name"
+  }
+}
```

Comment:
This object doesn't match with the object thats in our Postman collection. Have you tried to put this objects to an real EDC? Could you double check it?

\cc @tuncaytunc-zf

Reply:
Object in our postman collections looks like this:
```json
{
    "@context": {},
    "asset": {
        "@type": "Asset",
        "@id": "{{ASSET_ID}}",
        "properties": {
            "description": "Product EDC Demo Asset"
        }
    },
    "dataAddress": {
        "@type": "DataAddress",
        "type": "HttpData",
        "baseUrl": "https://jsonplaceholder.typicode.com/todos"
    }
}
```

Reply:
I've tried it, and it works. The only thing "wrong" there is the dataAddress.type, where I put a placeholder that doesn't look like one.